crates/assert-instr-macro/src/lib.rs

```diff
@@ -99,6 +99,7 @@ pub fn assert_instr(
     let shim_name_str = format!("{}{}", shim_name, assert_name);
     let to_test = quote! {
         #attrs
+        #[no_mangle]
```

Comment:
We probably should add a comment around here about which exact problem `#[no_mangle]` solves (debug-info weirdness).